client/js/helpers/contextMenu.ts

```diff
@@ -250,10 +250,12 @@ export function generateInlineChannelContextMenu(
 			switchToChannel(channel);
 		}
 
-		socket.emit("input", {
-			target: store.state.activeChannel.channel.id,
-			text: "/join " + chan,
-		});
+		if (store.state.activeChannel) {
+			socket.emit("input", {
+				target: store.state.activeChannel.channel.id,
+				text: "/join " + chan,
+			});
+		}
 	};
```

Comment:
how about some console.error or some such in the else path so that we notice something is off?